build.sbt

```diff
@@ -67,6 +67,9 @@ lazy val plugin = (project in file("plugin"))
         Seq("-Xmx1024M", "-Dplugin.version=" + version.value)
     },
     crossScalaVersions := Seq(scala212, scala3),
+    libraryDependencies ++= Seq(
+      "org.scalameta" %% "scalafmt-dynamic" % "3.9.9" cross CrossVersion.for3Use2_13,
+    ),
```

Comment:
Adding this dependency would end up creating a (tight) coupling with this specific version of Scalafmt, so if possible, I wonder if we can figure out some other way, like introducing an input task that takes a text file as argument on sbt-scalafmt, and call it dynamically if it is defined.